data/schema.ttl

```diff
@@ -4086,7 +4086,7 @@ See also the <a href="/docs/hotels.html">dedicated document on the use of schema
         :PostalAddress ;
     :rangeIncludes :Country,
         :Text ;
-    rdfs:comment "The country. For example, USA. You can also provide the two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1)." .
+    rdfs:comment "The country. For example, US. You can also provide the two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1)." .
```

Comment:
That is fair. The only concern with changing that wording is it fundamentally changes the definition as the way its worded currently almost feels like it can be either a country full text (USA, France, Greece) or the ISO compliant code alpha-2. If the expectation is it should always be the alpha-2 then the description needs to change to reflect that but could be breaking to some. Otherwise, I'm not sure if the ISO documentation provides any value outside of information that you may have not known in which it may be good to mention other codes such as the alpha-3 as being compliant as well.

This discussion may definitely be larger than I as previously thinking now after reading the wording.

Reply:
I tripped over this many times.

I suggest rephrasing it to:
"The country. It's recommended to use two-letter [ISO 3166-1 alpha-2 country code](http://en.wikipedia.org/wiki/ISO_3166-1). Do to historical reasons sometimes three-letter [ISO 3166-1 alpha-3 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3) or even full names e.g. United States, Germany, Singapore wear used."